packages/cms-cli/commands/create.js

```diff
@@ -18,14 +18,14 @@ const {
   addHelpUsageTracking,
 } = require('../lib/usageTracking');
 const { createFunctionPrompt } = require('../lib/createFunctionPrompt');
+const { createTemplatePrompt } = require('../lib/createTemplatePrompt');
 const { createModulePrompt } = require('../lib/createModulePrompt');
 const { commaSeparatedValues } = require('../lib/text');
 
 const COMMAND_NAME = 'create';
 
 const TYPES = {
   function: 'function',
-  'global-partial': 'global-partial',
```

Comment:
Not sure how often `global-partial` has been used but we should probably add handling that communicates to the user that they should use `hs create template` now and choose "global partial."